lib/api/model/model.g.dart

```diff
@@ -288,7 +288,9 @@ StreamMessage _$StreamMessageFromJson(Map<String, dynamic> json) {
     matchTopic: json['match_subject'] as String?,
     displayRecipient: json['display_recipient'] as String?,
     streamId: (json['stream_id'] as num).toInt(),
-  );
+  )..poll = Message._readPoll(json, 'submessages') == null
+      ? null
+      : Poll.fromJson(Message._readPoll(json, 'submessages'));
```

Comment:
Ow, this is inefficient. (Since `_readPoll` includes all the work of actually parsing the JSON.)

If there's not a quick and clean way to rearrange things to avoid this double parsing, that's fine and it shouldn't block merging this — polls aren't *that* common so it's not a major performance issue. But in that case let's leave a TODO comment about it at `Message.poll`.